deployment-examples/docker-compose/Dockerfile

```diff
@@ -59,7 +59,7 @@ COPY --from=builder /root/nativelink-bin /usr/local/bin/nativelink
 ARG ADDITIONAL_SETUP_WORKER_CMD
 
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends curl=8.5.0-2ubuntu10.6 \
```

Comment:
https://github.com/TraceMachina/nativelink/actions/runs/19063163878/job/54447370861 looks like a momentary network failure, not a failure to install a particular curl version, so these should be reverted